Microsoft.Toolkit/Collections/ReadOnlyObservableGroupedCollection.cs

```diff
@@ -52,8 +52,16 @@ private void OnSourceCollectionChanged(object sender, NotifyCollectionChangedEve
             // reporting the changes one by one. We consider only this case for now.
             if (e.OldItems?.Count > 1 || e.NewItems?.Count > 1)
             {
-                Debug.Fail("OldItems and NewItems should contain at most 1 item");
-                throw new NotSupportedException();
+                static void ThrowNotSupportedException()
+                {
+                    throw new NotSupportedException(
+                        "ReadOnlyObservableGroupedCollection<TKey, TValue> doesn't support operations on multiple items at once.\n" +
+                        "If this exception was thrown, it likely means support for batched item updates has been added to the " +
+                        "underlying ObservableCollection<T> type, and this implementation doesn't support that feature yet.\n" +
+                        "Please consider opening an issue in https://github.com/windows-toolkit/WindowsCommunityToolkit to report this.");
```

Comment:
@michael-hawker Could we use an aka.ms link to keep this from changing, just in case?

Reply:
💯 I should figure out if we have a link to the main repo, I know the /wct link just goes to the GitHub org...

https://aka.ms/windowstoolkit goes to the main repo, so we should use that. It was still pointing to the `Microsoft` org, so I've updated it. 🙂